Homework/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Tyrone Biggums</title>
+		<link rel="stylesheet" href="css/styles.css">
+		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+		<link rel="stylesheet" href="http://colorzilla.com/gradient-editor/#b4ddb4+0,83c783+17,52b152+33,008a00+67,005700">
+
+	</head>
+	<body>
+
+		<nav>
+			<a href="index.html">About Me |</a>
+			<a href="portfolio.html">Portfolio</a>
+			
+		</nav>
+
+		<h1 class="text-color">SHAZAM!!!</h1>
+		<hr>
```

Comment:
@JsonLitz Nothing wrong with using the `<hr>` tag, however, I almost never see them used in production. What you _could_ do is use an empty `<div>` with a border-top (or bottom) of 1px, something like this:

HTML:
`<div class="line"></div>`

CSS:
```
.line {
  border-top: 1px solid black;
}
```